py/ai/h2o/sparkling/ml/algos/H2OAutoML.py

```diff
@@ -64,18 +63,9 @@ def setParams(self, featuresCols=[], labelCol="label", allStringColumnsToCategor
 
         kwargs = get_input_kwargs(self)
 
-        validateEnumValues(self._H2OAutoMLParams__getAutomlAlgoEnum(), kwargs, "includeAlgos", nullEnabled=True)
-        validateEnumValues(self._H2OAutoMLParams__getAutomlAlgoEnum(), kwargs, "excludeAlgos", nullEnabled=True)
-        validateEnumValue(self._H2OAutoMLParams__getStoppingMetricEnum(), kwargs, "stoppingMetric")
-        validateEnumValue(self._H2OAutoMLParams__getSortMetricEnum(), kwargs, "sortMetric")
-
         if "projectName" in kwargs and kwargs["projectName"] is None:
```

Comment:
Shouldn't this be moved to parama spec as well?

Reply:
This is a bit special -> we are generating random string on Scala part, but that is not propagated to Python. But please see https://github.com/h2oai/sparkling-water/pull/1419 where I should improve this part

Reply:
After #1419 gets merged, should this be completely removed? Just wondering whether the constructor parameter and the setter will behave the same way.

Reply:
Yup, my bad #1419 should delete that, will fix it on that PR side